tests/inject/dynamic/shadow-dom.tests.ts

```diff
@@ -88,4 +88,67 @@ describe('Handle Shadow-DOM', () => {
         expect(getComputedStyle(shadowRoot.querySelector('p')).color).toBe('rgb(255, 26, 26)');
     });
 
+    it('should handle defined elements', async () => {
+        container.innerHTML = multiline(
+            '<svg-container>',
```

Comment:
You can remove the content inside the custom element, it is irrelevant to the test. Also rename `svg-container` to something neutral.